_test_/pet.test.js

```diff
@@ -1,11 +1,149 @@
 const Pet = require('../src/pet');
 
 describe('constructor', () => {
+  it('returns instance of an object', () => {
+    expect(new Pet('Romy')).toBeInstanceOf(Object);
+  });
+
   it('sets the name property', () => {
-    const pet = new Pet('Fido');
-    expect(pet.name).toEqual('Fido');
-    
     const romy = new Pet('Romy');
     expect(romy.name).toEqual('Romy');
   });
+
+  it('sets the initial age', () => {
+    const romy = new Pet('Romy');
+    expect(romy.age).toBe(0);
+  });
+
+  it('sets the growth in increament of 1', () => {
```

Comment:
mention what function you are testing in your test name as well as what each specific test. eg calling growUp() increments the growth by 1